qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -168,3 +171,72 @@ def study_prep_get_req(study_id, user_id):
     return {'status': 'success',
             'message': '',
             'info': prep_info}
+
+
+def study_files_get_req(study_id, prep_template_id, artifact_type):
+    """Returns the uploaded files for the study id categorized by artifact_type
+
+    It retrieves the files uploaded for the given study and tries to do a
+    guess on how those files should be added to the artifact of the given
+    type. Uses information on the prep template to try to do a better guess.
+
+    Parameters
+    ----------
+    study_id : int
+        The study id
+    prep_template_id : int
+        The prep template id
+    artifact_type : str
+        The artifact type
+
+    Returns
+    -------
+    dict of {str: object}
```

Comment:
The returned dictionary is keyed by strings, but the values to those keys can be either strings, int, or lists. In order to englobe all these types in the type description, I used the common base class `object`. Do you have a better alternative?